src/systems/tunnel-effect.js

```diff
@@ -74,7 +74,7 @@ AFRAME.registerSystem("tunneleffect", {
         Math.abs(r - TARGET_RADIUS) > CLAMP_RADIUS &&
         Math.abs(softness - TARGET_SOFTNESS) > CLAMP_SOFTNESS
       ) {
-        if (this.deltaR !== 0 && this.deltaS !== 0) {
+        if (!this.deltaR && !this.deltaS) {
           this.deltaR = TARGET_RADIUS - r;
           this.deltaS = softness - TARGET_SOFTNESS;
```

Comment:
I am still confused about why these are set once with some particular value for `r` and `softness`. Are `deltaR` and `deltaS` supposed to be constants? Do they need to be on `this`? Given their names, I expected them to change whenever `r` or `softness` changed, but perhaps they're better understood to be fixed increments by which we will change `r` or `softness` every frame. In that case, should these constants be defined above with `CLAMP_SPEED`, etc?

Reply:
the `deltaR` is defined by the value of `TARGET_RADIUS` and `minRadius`. Since we define the `minRadius` value in schema, I don't think it should be defined as a constant like `CLAMP_SPEED`.